pasta_eln/GUI/project.py

```diff
@@ -186,21 +187,17 @@ def projHeader(self) -> None:
 
   #TODO_P4 projectTree: select multiple items to edit... What is use case
   #TODO_P4 projectTree: allow right click on measurement to change recipe
-  def executeAction(self) -> None:
-    """ Any action by the buttons at the top of the page """
-    if hasattr(self.sender(), 'data'):  #action
-      menuName = self.sender().data()
-    else:                               #button
-      menuName = self.sender().accessibleName()
-    if menuName=='editProject':
+  def execute(self, identifier:list[Any]) -> None:
```

Comment:
Is there any specific reason not to use a multiple of 4 indentation which is quite a common standard used mostly?

Reply:
I use multiple of 2 indentation, which is another standard. I like it better and it is included in all the auto-formating: aka pylint.